legacy/gcc_5/Dockerfile

```diff
@@ -104,6 +104,11 @@ RUN dpkg --add-architecture i386 \
        && apt-get -qq update \
        && rm -rf /var/lib/apt/lists/*
 
+ARG CONAN_VERSION
```

Comment:
Moved CONAN_VERSION argument, because it represents a layer for Docker. So when we change the Conan version, we actually say to re-build every step after the parameter. However, we just want to install a new Conan version and nothing more, most of time.